--- android_project/brouter-core/src/main/java/btools/router/VoiceHintList.java
+++ android_project/brouter-core/src/main/java/btools/router/VoiceHintList.java
@@ -19,7 +19,7 @@

   private int transportMode = TRANS_MODE_BIKE;
   int turnInstructionMode;
-  List<VoiceHint> list = new ArrayList<>();
+  public List<VoiceHint> list = new ArrayList<>();

   public void setTransportMode(boolean isCar, boolean isBike) {
     transportMode = isCar ? TRANS_MODE_CAR : (isBike ? TRANS_MODE_BIKE : TRANS_MODE_FOOT);
