--- android_project/gradle/libs.versions.toml
+++ android_project/gradle/libs.versions.toml
@@ -62,6 +62,7 @@
 firebase-auth = { group = "com.google.firebase", name = "firebase-auth-ktx" }
 firebase-crashlytics = { group = "com.google.firebase", name = "firebase-crashlytics" }
 firebase-functions = { group = "com.google.firebase", name = "firebase-functions-ktx" }
+firebase-config = { group = "com.google.firebase", name = "firebase-config-ktx" }
 hilt-android = { group = "com.google.dagger", name = "hilt-android", version.ref = "hilt" }
 hilt-compiler = { group = "com.google.dagger", name = "hilt-android-compiler", version.ref = "hilt" }
 hilt-navigation-compose = { group = "androidx.hilt", name = "hilt-navigation-compose", version.ref = "hiltNavigationCompose" }
